src/coreclr/tools/aot/crossgen2.slnx

```diff
@@ -11,16 +11,24 @@
     <BuildType Solution="Checked|*" Project="Debug" />
   </Project>
   <Project Path="crossgen2/crossgen2.csproj">
-    <Platform Solution="*|Any CPU" Project="x86" />
+    <BuildDependency Project="ILCompiler.Diagnostics/ILCompiler.Diagnostics.csproj" />
```

Comment:
Are these project file changes intentional?

Reply:
Not intentional here, but I do have to keep modifying the slnx file to get VS builds to work. I'll submit a PR separately, though.